src/main/java/Duke.java

```diff
@@ -1,10 +1,75 @@
+
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner in = new Scanner(System.in);
+        List list = new List();
+
+
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?\n");
+
+
+        String userInput = in.nextLine();
+
+        while (!userInput.equals("bye")) {
+            if (userInput.equals("list")) {
+                list.printList();
+            }
+
+            else if(userInput.startsWith("done")){   //cannot use command.equals since user is entering more than just "done"
+                list.isComplete(userInput);
+            }
+
+
+            else if (userInput.startsWith("deadline")) {
+                list.addDeadline(getDeadlineDescription(userInput),getStringAfterByAt(userInput));
+            }
+
+
+            else if (userInput.startsWith("event")) {
+                list.addEvent(getEventDescription(userInput),getStringAfterByAt(userInput));
+            }
+
+            else if (userInput.startsWith("todo")) {
+                list.addTodo(userInput.substring(5));
+            }
+
+            else {
+                list.addTask(userInput);
+                System.out.println("added: " + userInput + "\n");
+            }
+
+            userInput = in.nextLine(); //read next user input
+        }
+
+        System.out.println("Bye. Hope to see you again soon!");
+
     }
+
+
+
+    public static String getDeadlineDescription (String input) {
+        String description = input.substring(9);
```

Comment:
You can maybe define a final variable for the value 9 that you used here. Avoid using magic number.